prow/gerrit/client/client_test.go

```diff
@@ -99,6 +99,35 @@ func TestQueryChange(t *testing.T) {
 			},
 			revisions: map[string][]string{},
 		},
+		{
+			name:       "one outdated change, but there's a new message",
```

Comment:
suggest add a testcase to assert the behavior for `merged change, with a new message`

Reply:
done